<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Orbital</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1420</width>
         <height>836</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_scrollArea">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="QFrame" name="frame_controlPanel">
           <layout class="QVBoxLayout" name="verticalLayout_controlPanel">
            <item>
             <widget class="QSplitter" name="splitter_controlPanel">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="childrenCollapsible">
               <bool>false</bool>
              </property>
              <widget class="QFrame" name="frame_controls">
               <layout class="QGridLayout" name="gridLayout_controls">
                <item row="0" column="1">
                 <widget class="QPushButton" name="pushButton_run">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Run</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QPushButton" name="pushButton_load">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Load Module</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QPushButton" name="pushButton_stop">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Stop</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QGroupBox" name="groupBox_messages">
               <property name="title">
                <string>Messages</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_messages">
                <item>
                 <widget class="QPlainTextEdit" name="plainTextEdit_messages">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QFrame" name="frame_params">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>1</verstretch>
                </sizepolicy>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_params">
                <item>
                 <widget class="QTableView" name="tableView_params"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QFrame" name="frame_plotPanel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_plotPanel">
            <item>
             <widget class="QScrollArea" name="scrollArea_plotPanel">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="widget_plotPanel">
               <layout class="QGridLayout" name="gridLayout_plots"/>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPlotEditor"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImportLogs">
   <property name="text">
    <string>Import Logs</string>
   </property>
  </action>
  <action name="actionImportConfig">
   <property name="text">
    <string>Import Filter Config</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionPlotEditor">
   <property name="text">
    <string>Plots</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
